### PR TITLE
[AdminBundle] Add hide sidebar configuration

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma separated list of tickets fixed by the PR

Add global option to hide sidebar

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -108,7 +108,8 @@ public function getConfigTreeBuilder()
                         ->integerNode('min_length')->defaultNull()->end()
                         ->integerNode('max_length')->defaultNull()->end()
                     ->end()
-            ->end();
+            ->end()
+            ->booleanNode('hide_sidebar')->defaultFalse()->end();
```
Comment (acrobat) [inline src/Kunstmaan/AdminBundle/DependencyInjection/Configuration.php]: Doesn't this need to be flipped? First the new option and than the closing end method?
```suggestion
                ->booleanNode('hide_sidebar')->defaultFalse()->end()
            ->end();
```

**Review by acrobat — CHANGES_REQUESTED** [review]
Can you also rebase against the latest 6.x branch, this should fix the failing tests? Thanks!

File: src/Kunstmaan/AdminBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (dannyvw) [reply src/Kunstmaan/AdminBundle/DependencyInjection/Configuration.php]: The end is from password_restrictions. I changed some indenting

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @dannyvw

**Pull request merged**